reference/build_helpers/visual_studio.rst

```diff
@@ -84,20 +84,22 @@ Builds Visual Studio project with the given parameters.
 
 Parameters:
     - **project_file** (Required): Path to the *.sln* file.
-    - **targets** (Optional, Defaulted to ``None``): List of targets to build.
-    - **upgrade_project** (Optional, Defaulted to ``True``): Will call :command:`devenv` to upgrade the solution to your current Visual Studio.
-    - **build_type** (Optional, Defaulted to ``None``): Use a custom build type name instead of the default ``settings.build_type`` one.
-    - **arch** (Optional, Defaulted to ``None``): Use a custom architecture name instead of the ``settings.arch`` one.
-      It will be used to build the ``/p:Configuration=`` parameter of :command:`MSBuild`.
-      It can be used as the key of the **platforms** parameter. E.g. ``arch="x86", platforms={"x86": "i386"}``
-    - **force_vcvars** (Optional, Defaulted to ``False``): Will ignore if the environment is already set for a different Visual Studio
-      version.
+    - **targets** (Optional, Defaulted to ``None``): Sets ``/target`` flag to the specified list of targets to build.
+    - **upgrade_project** (Optional, Defaulted to ``True``): Will call :command:`devenv /upgrade` to upgrade the solution to your current
+      Visual Studio.
+    - **build_type** (Optional, Defaulted to ``None``): Sets ``/p:Configuration`` flag to the specified value. Use a custom build type name
+      instead of the default ``settings.build_type`` one.
+    - **arch** (Optional, Defaulted to ``None``): Sets ``/p:Platform`` flag to the specified value. Use a custom architecture name instead
+      of the ``settings.arch`` one. It can be used as the key of the **platforms** parameter.
+      E.g. ``arch="x86", platforms={"x86": "i386"}``.
     - **parallel** (Optional, Defaulted to ``True``): Will use the configured number of cores in the :ref:`conan_conf` file or
       :ref:`tools_cpu_count`:
 
         - **In the solution**: Building the solution with the projects in parallel. (``/m:`` parameter).
-        - **CL compiler**: Building the sources in parallel. (``/MP:`` compiler flag)
-    - **toolset** (Optional, Defaulted to ``None``): Specify a toolset. Will append a ``/p:PlatformToolset`` option.
+        - **CL compiler**: Building the sources in parallel. (``/MP:`` compiler flag).
+    - **force_vcvars** (Optional, Defaulted to ``False``): Will ignore if the environment is already set for a different Visual Studio
+      version.
+    - **toolset** (Optional, Defaulted to ``None``): Sets ``/p:PlatformToolset`` to the specified toolset.
     - **platforms** (Optional, Defaulted to ``None``): Dictionary with the mapping of archs/platforms from Conan naming to another one. It
```

Comment:
- **platforms** (Optional, Defaulted to ``None``): This dictionary will update the default one, `msvc_arch` (see snippet below), and the resulting dictionary will be used to get the final value for the `/p:Platform` command line argument (see argument **arch** above).

...and I will explain an example using `arch` and `platforms` arguments with some values.